Source/WebCore/accessibility/AccessibilityTableCell.cpp

```diff
@@ -64,22 +64,20 @@ Ref<AccessibilityTableCell> AccessibilityTableCell::create(Node& node)
 
 bool AccessibilityTableCell::computeAccessibilityIsIgnored() const
 {
-    AccessibilityObjectInclusion decision = defaultObjectInclusion();
+    auto decision = defaultObjectInclusion();
     if (decision == AccessibilityObjectInclusion::IncludeObject)
         return false;
     if (decision == AccessibilityObjectInclusion::IgnoreObject)
         return true;
-    
+
     // Ignore anonymous table cells as long as they're not in a table (ie. when display:table is used).
-    RenderObject* renderTable = is<RenderTableCell>(renderer()) ? downcast<RenderTableCell>(*m_renderer).table() : nullptr;
+    auto* renderTableCell = dynamicDowncast<RenderTableCell>(renderer());
+    RenderObject* renderTable = renderTableCell ? renderTableCell->table() : nullptr;
```

Comment:
If I used auto* here the compiler would complain that node() was already deleted...

Reply:
Use `auto` and then call `element()` instead of `node()` below. `RenderElement::node()` is deleted because we know the associated Node is actually an Element and you're meant to call `RenderElement::element()` instead.